foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -166,11 +166,13 @@ inline Server<ServerConfiguration>::Server(std::string name, LogCallback logger)
 
   _server.clear_access_channels(websocketpp::log::alevel::all);
   _server.set_access_channels(APP);
-  _server.set_tcp_pre_init_handler(std::bind(&Server::socketInit, this, _1));
-  _server.set_validate_handler(std::bind(&Server::validateConnection, this, _1));
-  _server.set_open_handler(std::bind(&Server::handleConnectionOpened, this, _1));
-  _server.set_close_handler(std::bind(&Server::handleConnectionClosed, this, _1));
-  _server.set_message_handler(std::bind(&Server::handleMessage, this, _1, _2));
+  _server.set_tcp_pre_init_handler(std::bind(&Server::socketInit, this, std::placeholders::_1));
+  _server.set_validate_handler(std::bind(&Server::validateConnection, this, std::placeholders::_1));
+  _server.set_open_handler(std::bind(&Server::handleConnectionOpened, this, std::placeholders::_1));
+  _server.set_close_handler(
+    std::bind(&Server::handleConnectionClosed, this, std::placeholders::_1));
+  _server.set_message_handler(
+    std::bind(&Server::handleMessage, this, std::placeholders::_1, std::placeholders::_2));
```

Comment:
It's easier to read these using the `_1` `_2` shorthand, since we're using the std::placeholders namespace above.

Reply:
I had to use the full namespace specifier here as it would otherwise collide with boost placeholders. We include  (transitively) boost headers as ROS1 depends on it.

Reply:
I think we have the following options:
1. Remove the `using std::placeholders` statement and use the explicit namespace 
2. Move the server definition to a cpp file (would require explicit template instantiation for all `ServerConfiguration`s we want to support)

Reply:
Let's remove `using std::placeholders`. This is a header-only implementation since it will be merged into the upstream `ws-protocol` repo and act as a reference server implementation that can be copy-pasted into other C++ projects as a single file.